ui/pages/confirmations/components/confirm/scroll-to-bottom/scroll-to-bottom.tsx

```diff
@@ -31,16 +33,35 @@ const ScrollToBottom = ({ children }: ContentProps) => {
   const t = useContext(I18nContext);
   const dispatch = useDispatch();
   const currentConfirmation = useSelector(currentConfirmationSelector);
+  const previousId = usePrevious(currentConfirmation?.id);
 
   const {
     hasScrolledToBottom,
     isScrollable,
     isScrolledToBottom,
     onScroll,
     scrollToBottom,
+    setHasScrolledToBottom,
     ref,
   } = useScrollRequired([currentConfirmation?.id]);
 
+  /**
+   * Scroll to the top of the page when the confirmation changes. This happens
+   * when we navigate through different confirmations. Also, resets hasScrolledToBottom
+   */
+  useEffect(() => {
+    if (isEqual(previousId, currentConfirmation?.id)) {
```

Comment:
Any reason to use lodash isEqual to compate confirmationIds?

Reply:
oops. Didn't put too much thought into using it when I carried it over. No need and thanks for the catch! updated https://github.com/MetaMask/metamask-extension/pull/24435/commits/5f7bfaa30fced4ef266461244d6891ad1f6e4a2e